sass.py

```diff
@@ -325,6 +329,14 @@ def compile(**kwargs):
                                 output filename.  :const:`None` means not
                                 using source maps.  :const:`None` by default.
     :type source_map_filename: :class:`str`
+    :param source_map_contents: embed include contents in map
+    :type source_map_contents: :class:`bool`
+    :param source_map_embed: embed sourceMappingUrl as data URI
+    :type source_map_embed: :class:`bool`
+    :param omit_source_map_url: omit source map URL comment from output
+    :type omit_source_map_url: :class:`bool`
+    :param source_map_root: base path, will be emitted in source map as is
+    :type source_map_root: :class:`str`
```

Comment:
iirc we list these three times in the docstring for each "overload" of this function (yeah the interface isn't great -- I've been meaning to make a better public api but haven't gotten around to it) -- can you copy this block to the others as well?